config/webpack.config.prod.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
 const webpack = require('webpack')
+const {target} = require('./webpack.vars')
+const devtool = target === 'mobile' ? false : '#cheap-module-source-map'
 
 module.exports = {
   output: {
     filename: 'app.[hash].min.js'
   },
-  devtool: '#cheap-module-source-map',
+  devtool: devtool,
```

Comment:
I think we can totally disable sourcemaps in production, so explicitely set it to `false` here, whatever the context. I invoke @goldoraf @gregorylegarec @nono for their opinion about having sourcemaps for webapps in production.